docs/DeveloperGuide.md

```diff
@@ -943,7 +943,8 @@ testers are expected to do more *exploratory* testing.
        Expected: No contact is added due to duplicated phone number. Error details shown in the status message.
 
     1. Other incorrect add commands to try: `add <other valid params> b/<future date>`, `add <other valid params> p/hp:1200123`, `...`.
-
+       Expected: No contact is added. Error details shown in the status message.
```

Comment:
Should have one line between `Expected` and the previous line or include a \<br> at the end of the previous line, if not `Expected` will become part of the previous line.
<img width="1141" alt="Screenshot 2021-11-08 at 12 51 14 AM" src="https://user-images.githubusercontent.com/69960220/140654143-a15a80d0-3810-42f3-a420-c10cafe6dba9.png">